### ISSUE TITLE
Open Folder as Workspace doesn't work on Windows

### ISSUE THREAD

**Issue by tetchel** [microclimate-dev2ops/microclimate-vscode-tools#25]
```
[INFO: 17:32:28.870 OpenWorkspaceFolderCmd.js:62]: Opening folder file:///C/time/microclimate-1901/microclimate-1901/microclimate-workspace/microclimate-workspace/node
```

![image](https://user-images.githubusercontent.com/8888438/51570002-f60ea680-1e6b-11e9-93f6-48f1331f0d43.png)

**Comment by tetchel** [comment]
Fixed for Connections but not Projects

**Comment by tetchel** [comment]
Fixed in 19.1.1
